.gitignore

```diff
@@ -1 +1,6 @@
-node_modules
\ No newline at end of file
+/*.env
```

Comment:
instead of guiding ppl to deploy at the root dir, lets direct them to deploy at `deployments` dir and ignore it here